Users/Install.rst

```diff
@@ -147,27 +147,22 @@ Windows
 If you use windows, you can install docker easiest
 [here](https://www.docker.com/products/docker-toolbox).
 
-Installing coala from source
+Bleeding edge installation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. Warning::
+
+   This is only required for developers.
```

Comment:
In the issue description @Mixih said "Remove the install from git from the user docs and replace with a warning saying it is only required for devs and point them to the `pip3 install coala-bears --pre`". It's likely that I got it wrong. It would be then all right if I'll delete the warning?